feedparser/sanitizer.py

```diff
@@ -312,7 +325,22 @@ class HTMLSanitizer(BaseHTMLProcessor):
         "height",
         "letter-spacing",
         "line-height",
+        "margin",
+        "margin-bottom",
+        "margin-left",
+        "margin-right",
+        "margin-top",
+        "max-height",
+        "max-width",
+        "min-height",
+        "min-width",
+        "opacity",
```

Comment:
Now that I'm looking directly at these, it looks like these can be used to damage the rendering of the surrounding page, or to hide elements from being visible.